### ISSUE TITLE
How to map arrow keys in exwm-input-global-keys

### ISSUE THREAD

**Issue by t-hart** [ch11ng/exwm#597]
Hey,

I'm just getting started with EXWM and generally loving it so far, but I've hit a few issues with the documentation. In this case, it's specifically regarding using arrow keys in `exwm-input-global-keys`. 

First off, can you? And if so, how?

What I want to achieve is to map `s-right` and `s-left` to `next-buffer` and `previous-buffer` respectively. I tried a couple different approaches, but no luck:
```elisp
       ([?\s-right] . next-buffer)
       ([?\s-<right>] . next-buffer)
       ([?\<s-right>] . next-buffer)
```

In general, how the keys are interpreted as part of the `exwm-input-global-keys` function is unclear, as the format differs from the `set-key` function, for instance. Notably `M-SPC` is written out as `[?\M- ]`, which wasn't immediately obvious.

So in summary, for my immediate issue, how can I map arrow keys? And for a more general fix, how are the key combinations interpreted?

I'd be happy to contribute some documentation once I understand how things hang together, if that'd be helpful.

Cheers!

**Comment by ch11ng** [comment]
`exwm-input-global-keys` actually has a Customize interface that you can easily input keys by prefixing each of them with <kbd>C-q</kbd>.

The syntax for representing keys is not unique to EXWM but widely used elsewhere in Emacs.  Here a 'key sequence' is required and a key sequence can either be a string or a vector.  The `kbd` function returns key sequences, in your case

```
(kbd "s-<right>")
     ⇒ [s-right]
```

and it doesn't have a string representation.  Each element of a key sequence is a key event and can be retrieved with `(read-key)`.  Here are some resources that might be useful to you: `(info "(elisp)Basic Char Syntax")`, `(info "(elisp)Input Events")` and `(info "(elisp)Key Sequences")`.

**Comment by t-hart** [comment]
Thanks for the resources! I've used the `kbd` function a bit, but never seen this other way of specifying key sequences before. 

(Also, quite surprised that the `?\` escaping goes away when using the arrow keys, but if it works, it works.)

Thanks for the help and keep up the good work :100:
